data-loader/core/src/main/java/com/scalar/db/dataloader/core/dataimport/dao/ScalarDbStorageManger.java

```diff
@@ -0,0 +1,33 @@
+package com.scalar.db.dataloader.core.dataimport.dao;
+
+import com.scalar.db.api.DistributedStorage;
+import com.scalar.db.api.DistributedStorageAdmin;
+import com.scalar.db.service.StorageFactory;
+import java.io.IOException;
+import javax.annotation.Nullable;
+
+public class ScalarDbStorageManger {
+
+  @Nullable private final DistributedStorage storage;
+  private final DistributedStorageAdmin storageAdmin;
+
+  /**
+   * Class constructor
+   *
+   * @param storageFactory Factory to create all the necessary ScalarDB data managers
+   */
+  public ScalarDbStorageManger(StorageFactory storageFactory) throws IOException {
+    storage = storageFactory.getStorage();
+    storageAdmin = storageFactory.getStorageAdmin();
+  }
+
+  /** @return storage for ScalarDB connection that is running in storage mode */
```

Comment:
Is it repetitive but without doing this, a warning appears when compiling the code. Evenb if you follow the Google style guide recommendation, another warning is triggered.
```
 warning: [MissingSummary] A summary fragment is required; consider using the value of the @return block as a summary fragment instead.
   * @return Distributed Transaction manager for ScalarDB connection that is running in transaction
     ^
    (see https://google.github.io/styleguide/javaguide.html#s7.2-summary-fragment)
  Did you mean '*Returns distributed Transaction manager for ScalarDB connection that is running in transaction'?

```

```suggestion
  Returns storage for ScalarDB connection that is running in storage mode
  /** @return storage for ScalarDB connection that is running in storage mode */
```